copybookreader-plugins/pom.xml

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 2015 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <artifactId>hydrator-plugins</artifactId>
+    <groupId>co.cask.hydrator</groupId>
+    <version>1.4.0-SNAPSHOT</version>
+  </parent>
+
+  <name>CopybookReader Plugins</name>
+  <artifactId>copybookreader-plugins</artifactId>
+  <modelVersion>4.0.0</modelVersion>
+
+  <properties>
+    <jrecord.version>0.80.8</jrecord.version>
+    <cb2xml.version>0.80.8-jrecord</cb2xml.version>
+    <main.basedir>${project.basedir}</main.basedir>
+    <project-version>3.5.0-SNAPSHOT</project-version>
+  </properties>
+
+  <repositories>
```

Comment:
We are using Jrecord jars from the local repo as is not available in maven repo, and therefore have to specify the version used.